1.8/administration/user-titles.md

```diff
@@ -0,0 +1,61 @@
+---
+layout: page
+title:  "User Titles"
+categories: [administration]
+---
+
+## About User Titles
+
+User titles are text displayed under the username in posts, threads, profile and memberlist.
+Each group have it's own default user title wich is shown if user don't use his custom one.
+
+## How to set Default User Titles
+
+You can set them to be default for one user group or to be changed with number of posts what user made.
+
+#### Set User Titles According the Group
+
+At `Admin cp -> Users & Groups -> Groups -> <usergroup name>` You can give default usertitle for that group.
+
+Just fill **Default User Title** filed.
+
+#### Set User Titles According the Number of Users Posts
+
+Group Registered by default have this option.
+If you want to use that option, leave Default User Title field in group settings blank.
+
+You can specify and edit number of posts, names or add new Custom User Titles at `Admin cp -> Users & Groups -> User Titles`.
+
+When adding or editing them, you have 4 options: Title to Assign, Minimum posts, Number of Stars and Star image.
+
+##### Title to Assign field
+
+Here write text for Custom User Title
+
+##### Minimum Posts
+
+Here write minimum number of posts, requied before user got assigned this Custom User Title.
+
+##### Number of Stars
+
+Here you should put number of stars wich will be shown under user title. If you don't want stars, you will need to put 0 to that filed.
+
+##### Star image
+
+Here you should put star image url. That image will be shown like star in profiles.
+
+## Edit Custom User Title for Specific User
```

Comment:
Oh, that. I added it, to be easier to navigate, and after that 3 ways to do that. That 3 ways are this sublevels of that... Subtitles. Fell free to edit if you think that it should be different.